service/src/main/resources/application.properties

```diff
@@ -2,7 +2,7 @@ spring.datasource.url=
 spring.datasource.username=
 spring.datasource.password=
 spring.datasource.type=com.zaxxer.hikari.HikariDataSource
-spring.mvc.throw-exception-if-no-handler-found=true
+spring.mvc.throw-exception-if-no-handler-found=false
```

Comment:
If this is set to true the response is a 500.  With it set to false a 404 is returned.  To me the latter is more appropriate because it isn't a server error if you try to hit a URL that doesn't exist in our system.